exp-models/addon/models/session.js

```diff
@@ -18,6 +18,9 @@ export default DS.Model.extend(AnonJamModel, {
     experimentId: DS.attr('string'),
     experimentVersion: DS.attr('string'),  // TODO: Currently this field is not acted on in any way
 
+    frameIndex: DS.attr(),
```

Comment:
Also, at minimum, a source code comment explaining the purpose of these parameters would be helpful, for anyone trying to understand what these fields are later.

(given how we approach pagination in this app, the meaning of `frameIndex` is especially murky. Page number? Number of pages? Number of the subframe within the frame within the page pretending to be 10 frames?)